ad-insertion/analytics/VCAC-A/gst/Dockerfile

```diff
@@ -0,0 +1,32 @@
+# ssai_analytics_gst_vcaca 
+
+FROM openvisualcloud/vcaca-ubuntu1804-analytics-gst:19.11
+
+# Fetch python3 and Install python3
+RUN DEBIAN_FRONTEND=noninteractive apt-get update && apt-get install -y -q --no-install-recommends libjson-c3 python3-gst-1.0 python3-jsonschema python3-gi python3-requests python3-tornado python3-pip python3-setuptools python3-wheel &&  rm -rf /var/lib/apt/lists/*;
+
+#COPY ./app/server/requirements.txt /
+#RUN pip3 install  --no-cache-dir -r /requirements.txt -i https://pypi.tuna.tsinghua.edu.cn/simple && \
+Run pip3 install kafka-python==1.3.2 -i https://pypi.tuna.tsinghua.edu.cn/simple && \
+    pip3 install kazoo==2.2.1  -i https://pypi.tuna.tsinghua.edu.cn/simple
```

Comment:
Why not use apt-get install python3-kafka and python3-kazoo? pip introduces the entire gcc tool chain into the final image. It's not a good tool for building docker images.

Reply:
update with python3-kafka and python3-kazoo